src/test/scala/temple/TestUtils.scala

```diff
@@ -24,7 +24,10 @@ object TestUtils {
           FileVisitResult.CONTINUE
 
         override def visitFile(file: Path, attrs: BasicFileAttributes): FileVisitResult = {
-          foundFiles += File(path.relativize(file).getParent.toString, file.getFileName.toString) -> FileUtils.readFile(
+          foundFiles += File(
+            Option(path.relativize(file).getParent).map(_.toString).getOrElse(""),
```

Comment:
When does this return `null`?

Reply:
When the `file` you're relativizing to is in the same directory as `path`,  `getParent` is `null`

Reply:
Nice that 😉